test/parallel/test-tls-client-getephemeralkeyinfo.js

```diff
@@ -1,20 +1,21 @@
 'use strict';
 const common = require('../common');
+const commonFixturesDir = require('../common.fixtures').fixturesDir;
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
 const assert = require('assert');
 const tls = require('tls');
 const fs = require('fs');
 
-const key = fs.readFileSync(`${common.fixturesDir}/keys/agent2-key.pem`);
-const cert = fs.readFileSync(`${common.fixturesDir}/keys/agent2-cert.pem`);
+const key = fs.readFileSync(`${commonFixturesDir}/keys/agent2-key.pem`);
+const cert = fs.readFileSync(`${commonFixturesDir}/keys/agent2-cert.pem`);
```

Comment:
This should actually use the `fixtures.readKey` function. I fix this while landing.

Reply:
@timothywisdom it would actually be nice if you could fix this test.